### PR TITLE
Remove option java_package from protobuf file.

### PR DESCRIPTION
**Pull request by maxlambrecht**
We are not using the option `java_package` from the protobuf definition, so better remove it.
 
Signed-off-by: Max Lambrecht <maxlambrecht@gmail.com>

### CODE REVIEW

File: proto/workload.proto [hunk 1]
```diff
@@ -1,7 +1,5 @@
 syntax = "proto3";
 
-option java_package = "io.spiffe.workloadapi.grpc";
```
Comment (Andres-GC) [inline proto/workload.proto:3]: Good Catch Max. 
This doesn't require the *_pb files to be regenerated?
If it is not necessary but resulting files are different, them I'm in favor of generate the new ones and update the code.

Comment (maxlambrecht) [reply proto/workload.proto:3]: Sorry, missed that. It does change, doesn't affect anything but still worthwhile to keep them in sync. 
Thanks for the catch.

**Review by Andres-GC — APPROVED** [review]
Great Max :medal_military:

**Review by mchurichi — APPROVED** [review]
LGTM

**Pull request merged**